internal/policy/policy_test.go

```diff
@@ -55,6 +56,9 @@ func (c *FakeKubernetesClient) FetchEnterpriseContractPolicy(ctx context.Context
 }
 
 func TestNewPolicy(t *testing.T) {
+	timeNowStr := "2022-11-23T16:30:00Z"
+	timeNow, _ := time.Parse(time.RFC3339, timeNowStr)
```

Comment:
For our own future sanity, can we fail the test if `time.Parse` does return an error?

Reply:
8e42aa0b6b4e1aa55dab15dbdfdb63b822399a04